src/collection-preferences/index.tsx

```diff
@@ -65,39 +65,43 @@ const ModalContent = ({
     );
   }
 
+  const hasLeftContent = !!(pageSizePreference || wrapLinesPreference || stripedRowsPreference || customPreference);
```

Comment:
This is also defined below in CollectionPreferences. Can it not be accepted as a prop to avoid duplicated logic?

Reply:
We can.. I just felt like it's too trivial to add a prop for, since it's something you can calculate based on the existing props.